lib/models/addon.js

```diff
@@ -1243,6 +1232,26 @@ let addonProto = {
     });
   },
 
+  /**
+    Returns a tree with tests output given tree. if tree name is not passed it will use type to lookup tree
+    @private
+    @param  {String} type will be fowarded to addon as type paramter, and is used to find path
+    @param  {Tree} treeName optional if this paramter is passed it will not lookup tree using treeName, and pass this tree instead
+    @method _lintTree
+    @return {Tree} Tree with linter tests
+  */
+  _lintTree(type, treeName, destDir) {
+    let tree = null;
+    treeName = treeName || type;
+    destDir = destDir || treeName;
```

Comment:
we don't seem to use the `treeName` or `destDir` parameters anywhere when calling the method. IMHO we shouldn't add them unless we actually need them.